UKHomeOffice/sgmr-service
# SPCR-509 Add NotificationBanner to /people

### AC / Description of changes
The NotificationBanner component is missing from the /people page therefore, when you delete a person the success notification banner shows on /voyage page. 

Add the NotificationBanner to the people page so that the success banner appears correctly on the /people page after deleting a person

### To Test
- On the /people page
- If a saved person is present, delete the person
- (If no saved people, create one and then delete them)
- > The person successfully deleted banner should show on the /people page
- > This banner should NOT be present on the /voyage page

### Notes

The tests were failing after adding the banner - to stop the error, the value of the NotificationBanner has been set to null (could not find a better way to stop the tests failing)

---

**Remember to merge to feature branches not `master`.**
